New compile issue with V2 makefile

With the changes to the V2 makefile for Mac (in compile/DYLIB/), I am getting compile issues with Waves.h:

../../source/Waves/../Waves.h:68:13: error: expected ';' after top level declarator
        int DECLDIR MoorDyn_GetWavesKin(MoorDynWaves waves,
                   ^
                   ;
../../source/Waves/../Waves.h:85:9: error: redefinition of 'DECLDIR' with a different type: 'double' vs 'int'
        double DECLDIR WaveNumber(double Omega, double g, double h);

I am not super familiar with how the new makefile works, and can't figure out why the compiler is not recognizing DECLDIR.

--- a/compile/DLL/makefile
+++ b/compile/DLL/makefile
@@ -59,7 +59,7 @@ Waves/%.o: ../../source/Waves/%.cpp $(HEADERS)
 	${CXX} $(CPPFLAGS) -o $@ $<
 
 clean:
-	$(RM) *.exe.manifest *.exe *.obj *.o *.dll
+	$(RM) *.exe.manifest *.exe *.obj *.o ./*/*.o *.dll
 
 test_minimal.exe: ../../tests/minimal.cpp libmoordyn2.dll
 	$(CXX) $(EXEFLAGS) -L./ -lmoordyn2 -I../../source/ -o test_minimal.exe ../../tests/minimal.cpp

--- a/compile/DYLIB/makefile
+++ b/compile/DYLIB/makefile
@@ -68,7 +68,7 @@ Waves/%.o: ../../source/Waves/%.cpp $(HEADERS)
 	${CXX} $(CPPFLAGS) -o $@ $<
 
 clean:
-	rm -f *.o *.dylib *.dylib.*
+	rm -f *.o ./*/*.o *.dylib *.dylib.*
 	rm -f test_*
 
 test_minimal: ../../tests/minimal.cpp libmoordyn2.dylib

--- a/compile/SO/makefile
+++ b/compile/SO/makefile
@@ -64,7 +64,7 @@ Waves/%.o: ../../source/Waves/%.cpp $(HEADERS)
 	${CXX} $(CPPFLAGS) -o $@ $<
 
 clean:
-	rm -f *.o *.so *.so.*
+	rm -f *.o ./*/*.o *.so *.so.*
 	rm -f test_*
 
 test_minimal: ../../tests/minimal.cpp libmoordyn.so

--- a/source/Waves.h
+++ b/source/Waves.h
@@ -35,6 +35,8 @@
 #ifndef WAVES_H
 #define WAVES_H
 
+#include "MoorDynAPI.h"
+
 #ifdef __cplusplus
 extern "C"
 {

